tests/Microsoft.Graph.Core.Test/Requests/HttpProviderTests.cs

```diff
@@ -82,6 +82,20 @@ public void HttpProvider_DefaultConstructor()
             }
         }
 
+        [TestMethod]
+        public void HttpProvider_HttpMessageHandlerConstructor()
+        {
+            using (var testhttpMessageHandler = new TestHttpMessageHandler())
```

Comment:
line 88: let's remove it. Setup() creates a TestHttpMessageHandler for us. 
line 89 and 92: the argument should now be this.testHttpMessageHandler 

This PR fixes #285.

Reply:
Same fix applies to the DotNetCore tests. Other than that these nitpicks, this is all good.